v2/googlecloud-to-neo4j/src/main/java/com/google/cloud/teleport/v2/neo4j/DataFlowJvmStart.java

```diff
@@ -0,0 +1,31 @@
+package com.google.cloud.teleport.v2.neo4j;
+
+import com.google.auto.service.AutoService;
+import java.security.Security;
+import org.apache.beam.sdk.harness.JvmInitializer;
+import org.apache.beam.sdk.options.PipelineOptions;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This class will be picked up and used at the very start of a DataFlow JVM. As such it allows us
+ * to disable certain security algorithms so that secure connections automatically pick up the right
+ * one. This is an issue for secure Kafka and Neo4j Aura connections (neo4j+s:// protocol).
+ */
+@AutoService(value = JvmInitializer.class)
+public class DataFlowJvmStart implements JvmInitializer {
```

Comment:
We now have a common JvmInitializer here:
https://github.com/GoogleCloudPlatform/DataflowTemplates/blob/main/v2/common/src/main/java/com/google/cloud/teleport/v2/common/CommonTemplateJvmInitializer.java
Which will conflict with this DataFlowJvmStart.

I would simply remove DataFlowJvmStart and let the template users control the algorithms through CommonTemplateJvmInitializer and the "disabledAlgorithms" param.

Having the "common" module in the dependencies (which you already have) should be enough to activate CommonTemplateJvmInitializer.

If you really want to hard-code the list of disabled algorithms I'd do it in the template code:
```
options.setDisabledAlgorithms(...);
final GcpToNeo4j bqToNeo4jTemplate = new GcpToNeo4j(options);
```

Otherwise 2 initializers will be conflicting with each other, e.g. the order of their execution is not guaranteed, it's possible you'll override the algorithms in DataFlowJvmStart but then CommonTemplateJvmInitializer will overwrite the value.

Reply:
I'm not sure what pattern to use.  This method expects a ValueProvider<String> which cannot be supplied programmatically:

```
CommonTemplateOptions commonPipelineOptions = options.as(CommonTemplateOptions.class);
        commonPipelineOptions.setDisabledAlgorithms("SSLv3, RC4, DES, MD5withRSA, DH keySize < 1024, EC keySize < 224, 3DES_EDE_CBC, anon, NULL");
```

Reply:
I was including the legacy not v2 version of CommonTemplate options... works now.